include/mxnet/c_api.h

```diff
@@ -1582,6 +1660,22 @@ MXNET_DLL int MXSymbolInferShape(SymbolHandle sym,
                                  const mx_uint ***aux_shape_data,
                                  int *complete);
 
+MXNET_DLL int MXSymbolInferShape64(SymbolHandle sym,
```

Comment:
Is there an implementation of this function? If no, maybe we don't need to declare it.

Reply:
this is only a declaration. Its implementation is not yet there, I will add implementation with subsequent commits